code/game/objects/items/contraband_vr.dm

```diff
@@ -36,12 +36,12 @@
 						/obj/item/device/pda/mime,
 						/obj/item/device/pda/syndicate,
 						/obj/item/mecha_parts/chassis/phazon,
-						/obj/item/mecha_parts/part/phazon_head,
-						/obj/item/mecha_parts/part/phazon_left_arm,
-						/obj/item/mecha_parts/part/phazon_left_leg,
-						/obj/item/mecha_parts/part/phazon_right_arm,
-						/obj/item/mecha_parts/part/phazon_right_leg,
-						/obj/item/mecha_parts/part/phazon_torso,
```

Comment:
I know the odds are **extremely** slim. That's the point. Not all of the items you get from the stolen goods crate are useful.

Though phazon parts are still worth juicy research points.